helpers/deploymentConfig.ts

```diff
@@ -69,6 +69,7 @@ export const DEFAULT_STALE_PERIOD = 24 * 60 * 60; // 24 hrs
 const STALE_PERIOD_100M = 60 * 100; // 100 minutes (for pricefeeds with heartbeat of 1 hr)
 const STALE_PERIOD_26H = 60 * 60 * 26; // 26 hours (pricefeeds with heartbeat of 24 hr)
 export const ANY_CONTRACT = ethers.constants.AddressZero;
+export const PENDLE_TWAP_DURATION = 1800;
```

Comment:
I think the TWAP Duration will be an item in the `OracleConfig` configuration, to be able to define it per asset. But we can keep it here at the moment